scripts/go-mod-tidy.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
 go mod tidy
-if ! git diff --exit-code go.mod go.sum ; then
+if ! git diff --exit-code go.mod; then
+    echo -e "\nPlease commit the changes made by 'go mod tidy'"
+    exit 1
+fi
```

Comment:
Is this necessary?

Reply:
There was an issue with the version of `git` installed in the Mariner image. I _think_ I fixed this elsewhere in CI, so I'll try reverting this change